geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisStringTest.java

```diff
@@ -351,4 +354,50 @@ public void setExpirationTimestamp_stores_delta_that_is_stable() throws IOExcept
     o2.fromDelta(in);
     assertThat(o2).isEqualTo(o1);
   }
+
+  @Test
+  public void overheadConstants_shouldNotChange_withoutForethoughtAndTesting() {
+    assertThat(RedisString.PER_OBJECT_OVERHEAD).isEqualTo(8);
+    assertThat(RedisString.getPerStringOverhead()).isEqualTo(RedisString.PER_STRING_OVERHEAD);
```

Comment:
The `getPerStringOverhead` method just returns the same constant we are checking against.  Could we test this constant using the ROS size of `RedisString` like we did for `RedisHash`?  Also, could we change the name of the test to something like `constantRedisStringOverhead_shouldEqual_calculatedOverhead` ?